Resources/Prototypes/Entities/Mobs/NPCs/revenant.yml

```
@@ -14,7 +14,7 @@
   - type: Sprite
     noRot: true
     drawdepth: Ghosts
-    sprite: DeltaV/Mobs/Ghosts/revenant.rsi # DeltaV - Custom revenant sprite
+    sprite: Mobs/Ghosts/revenant.rsi # DeltaV - Custom revenant sprite
```

```suggestion
    sprite: Mobs/Ghosts/revenant.rsi
```
While this changes works, some maintainers wishes to keep the previous deltaV one then wizden one.